episodes/infrastructure.md

```diff
@@ -214,12 +214,15 @@ it is safest to replace it with another value within quotation marks.
 
 ### Practice with `config.yaml` (5 minutes)
 
-Complete the configuration of your lesson by adjusting the following fields in `config.yaml`:
+Complete the configuration of your lesson by adjusting the following fields in `config.yaml`. 
+Remember that FIXME values with quotes should be replaced with another value still in quotes and 
+FIXMEs without quotes should be replaced by values without quotes.
 
-- `email`: add an email address people can contact with questions about the lesson/project.
+- `contact`: add an email address people can contact with questions about the lesson/project.
 - `created`: the date the lesson was created (today's date) in YYYY-MM-DD format.
 - `keywords`: a (short) list of keywords for the lesson,
```

Comment:
```suggestion
- `keywords`: a (short) comma-separated list of keywords for the lesson,
```

Reply:
I like this clarification, while the section is already being updated in this PR.